CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/QualityControlViewPane.java

```diff
@@ -314,11 +317,13 @@ public void updateItem(final QualityControlEvent item, final boolean empty) {
                 };
 
                 cell.setOnMouseClicked(value -> {
-                    if (value.getClickCount() == 2) {
+                    if (value.getClickCount() == 2 && !QualityControlViewTopComponent.isFirstClick() || 
+                            value.getClickCount() > 2 && !QualityControlViewTopComponent.isFirstClick()) {
```

Comment:
The `if` conditions can be simplified to:
```
        if (value.getClickCount() >= 2 && !firstClick) {
```
And have `firstClick` defined as a local class member variable.